client/src/views/OrgsList.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { NavLink } from 'react-router-dom';
+import Typography from '@mui/material/Typography';
+import makeStyles from '@mui/styles/makeStyles';
+import Card from '@mui/material/Card';
+import TodayOutlined from '@mui/icons-material/TodayOutlined';
+import RoomOutlined from '@mui/icons-material/RoomOutlined';
+import Grid from '@mui/material/Grid';
+
+import type { Theme } from '@mui/material/styles';
+
+import type { Asset } from '../types';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  needsAndOffersSub: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    flexWrap: 'wrap',
+    alignItems: 'stretch',
+  },
+  card: {
+    marginBottom: '30px',
+    '&:not(:last-child)': {
+      marginRight: '20px',
+    },
+    flex: '0 0 30%',
+  },
+  cardImg: {
+    borderRadius: '5px',
+    margin: '10%',
+    maxWidth: '80%',
+  },
+  needsAndOffersHeader: {
+    textAlign: 'left',
+    paddingBottom: '20px',
+  },
+  needsAndOffers: {
+    padding: '10%',
+  },
+  cardText1: {
+    padding: '0 10%',
+  },
+  cardText2: {
+    padding: '0 10% 10%',
+  },
+}));
+
+type Props = {
+  assets: Asset[];
```

Comment:
orgs: Organization[]

Reply:
same

Reply:
OrgsList.tsx
![Screenshot 2022-11-04 at 7 28 59 PM](https://user-images.githubusercontent.com/92830192/200097157-6b778edf-892a-4cf6-95d8-3aa4aa58a481.png)

I was looking at commit [6fa2bbc](https://github.com/Nonprofit-Exchange-Hub/web-app/pull/230/commits/6fa2bbc334d5b86258dfed19363d727991761c32) 8 days ago. Let me know if you see something else and which commit it is. I'll fix it if so.